src/main/java/io/github/mooy1/simpleutils/implementation/blocks/Elevator.java

```diff
@@ -57,8 +57,14 @@ public boolean isSynchronized() {
             @Override
             public void tick(Block block, SlimefunItem slimefunItem, Config config) {
                 if (SimpleUtils.slimefunTickCount() % 16 == 0 && block.getY() > block.getWorld().getMinHeight()) {
+                    final List<Material> blackList = List.of(
```

Comment:
Additionally, you have not imported `List` so it's failing to build, this leads me to also conclude this has not been built and tested locally?

Reply:
Sorry but is not List being imported on line 3?

https://github.com/RenanMsV/SimpleUtils/blob/0d3db22b0024a151596da0d683cf7488e2045b13/src/main/java/io/github/mooy1/simpleutils/implementation/blocks/Elevator.java#L3

Reply:
Ha you're right, I may need to get my eyes checked sorry, in which case it's the `.of` it's not liking likely due to the build version so best to declare and init separately, feel free to use a static block as required. I'll look into the build version separately and change back to using the of method if appropriate.

Reply:
I used the .of method since it was simpler and didnt require importing another class.
To try to fix this I will use `Arrays.asList` from `java.util.Arrays` which will require adding a new import